app/celery/service_callback_tasks.py

```diff
@@ -0,0 +1,71 @@
+import json
+from app import (
+    DATETIME_FORMAT,
+    notify_celery,
+)
+from app.dao.notifications_dao import (
+    get_notification_by_id,
+)
+
+from app.statsd_decorators import statsd
+from app.dao.service_callback_api_dao import get_service_callback_api_for_service
+from requests import (
+    HTTPError,
+    request,
+    RequestException
+)
+from flask import current_app
+from app.config import QueueNames
+
+
+@notify_celery.task(bind=True, name="send-delivery-status", max_retries=5, default_retry_delay=300)
+@statsd(namespace="tasks")
+def send_delivery_status_to_service(self, notification_id):
+    # TODO: do we need to do rate limit this?
+    notification = get_notification_by_id(notification_id)
+    service_callback_api = get_service_callback_api_for_service(service_id=notification.service_id)
+    if not service_callback_api:
```

Comment:
We should probably check for the service_callback_api before creating this task.

Reply:
Agree. I will leave this check here as we need to get service_callback_api anyway. But will add the check before putting the task on queue.